lib/tmuxinator/window.rb

```diff
@@ -10,6 +10,7 @@ def initialize(window_yaml, index, project)
               end
       @root = nil
       @panes = []
+      @commands = []
```

Comment:
Should we add a test for this, that verifies a valid `commands` instance, given an empty `panes` in the yaml?